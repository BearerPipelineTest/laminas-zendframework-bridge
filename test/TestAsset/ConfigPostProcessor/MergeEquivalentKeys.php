<?php
return [
    'zf-rest' => [
        [
            'name' => 'Some\Rest\Controller',
        ],
    ],
    'zend-expressive' => null,
    'expressive' => [
        'error_handler' => [
            'template_404' => 'custom::404',
            'template_error' => 'custom::500',
        ],
    ],
    'apigility-rest' => 'Some\Other\Rest\Controller',
];
