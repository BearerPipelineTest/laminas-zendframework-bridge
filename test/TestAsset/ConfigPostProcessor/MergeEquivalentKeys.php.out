<?php
return [
    'apigility-rest' => [
        [
            'name' => 'Some\Rest\Controller',
        ],
        'Some\Other\Rest\Controller',
    ],
    'expressive' => [
        'error_handler' => [
            'template_404' => 'custom::404',
            'template_error' => 'custom::500',
        ],
    ],
];

